templates/web/base/waste/garden/modify.html

```diff
@@ -21,6 +21,7 @@
     data-per_new_bin_cost="[% per_new_bin_cost %]"
     data-pro_rata_bin_cost="[% garden_form_data.pro_rata_bin_cost %]"
     [% IF is_staff && waste_features.ggw_discount_as_percent %]
+        data-discount_applied="[% form.saved_data.apply_discount %]"
```

Comment:
What does the new subscription page do, which also asks for the discount before the page with the numbers/prices on? Looks like it adjusts per_bin_cost so then the already-adjusted value is in the data here, meaning it doesn't need an extra flag. 
This is okay for getting it on staging for them to look at, but after we should probably decide which of two methods is clearer and go with that (think it might be this one, though, this extra flag feels a bit clearer maybe?)

Reply:
Ok, I've put it on staging with this as is, but removed the on-change in js file as suggested below.

Reply:
@dracos This is what the new fixup relates to - stop the discount adjusting the new subscriptions